api.swagger.yaml

```diff
@@ -12218,6 +12218,22 @@ components:
           description: The subject of the notification
           type: string
           example: mentioned you in a post
+        target:
```

Comment:
Have a look in swagger at "discriminator". There needs to be a way for schema validation to determine what the type is.